README.md

```diff
@@ -1,88 +1,130 @@
 # Akka Serialization Helper
 
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.virtuslab/sbt-akka-serialization-helper/badge.svg)](https://maven-badges.herokuapp.com/maven-central/cz.jirutka.rsql/rsql-parser)
+
+
```

Comment:
Pls include logo while you're here... https://github.com/VirtusLab/git-machete/blob/master/README.md for a reference HTML tag FWIW